packages/react-native/React/CoreModules/RCTPerfMonitor.mm

```diff
@@ -84,6 +84,12 @@ @interface RCTPerfMonitor : NSObject <
 
 @end
 
+#if __has_include(<React/RCTDevMenu.h>)
+@interface RCTPerfMonitor() <DevMenuItemFactory>
+
+@end
+#endif
```

Comment:
I we remove the `DevMenuItemFactory`, this can go. This will also avoid a bunch of possible irritating issues due to header search paths and similar...